content/rules.md

```diff
@@ -0,0 +1,105 @@
+
+### Business teams
+
+A business team:
+
+- Consists up to 3 persons who are employed by the same company or institution.
+- Participates in the business teams pool for the title "Winner of the Benelux Algorithm Programming Contest 2021".
+
+## The Contest
+
+### Introduction
+
+- The language of the contest is English. All written contest materials will be in English.
+- The contest lasts for 5 hours.
+- From the beginning until one hour before the end of the BAPC, the scores are displayed.
+
+### Problems
+
+- At least six problems will be posed. The jury intents to develop problems with as little as possible dependence on detailed knowledge of a particular applications area or a particular contest language.
+- A contestant may submit a claim of ambiguity or error in a problem statement by submitting a clarification request to a judge. If the judges agree that an ambiguity or error exists, a clarification will be issued to all contestants.
+- The jury has the right to change or withdraw problems during the contest before informing all teams.
+
+### System
+
+- A solution for a problem has to be written in C99, C++ (version 14 or later will be available), Python (versions 3 will be available), Java (version 11 or later) or Kotlin (version 1.3), unless the problem statement explicitly states otherwise. Business teams can request other submission languages, on which the organization will decide whether it will be made available for submission.
+- A team is disqualified if they submit harmful code that disrupts the contest
+- A submission can use the standard library of the language.
+- A list of usable libraries are provided by the organization before the contest.
+- On-site contest
+  - Each team will use at most one workstation.
+  - There is an explicit restriction on internet access during the contest. Interaction about problems is allowed via clarification requests with the jury and internal discussion within the team. Asking questions about the contest to crew or coaches is subject to disqualification.
+- Online contest
+  - Each team will use at most three workstations, i.e. at most one computer per team member.
+  - There is no explicit restriction on internet access during the contest. The organization discourages copying code from reference materials as solutions will be checked on similarities. If there is suspicion of plagiarism among teams, both teams are subject to disqualification.
+  - Interaction about problems is allowed from clarification requests with the jury and internal discussion within the team. Asking questions about the contest to outside parties such as sites such as Stack Overflow are not allowed. The same applies to general programming errors.
+
+### Judgment
+
+- Each run is judged as accepted or rejected by a judge, and the team is notified of the results.
+- For each problem, the jury has a correct solution and test data.
```

Comment:
```suggestion
- For each problem, the jury has the correct solution and test data.
```

Reply:
Something can have multiple solutions (multiple strategies) so I think there is no highlander situation here.